src/d3-org-chart.js

```diff
@@ -72,11 +74,15 @@ export class OrgChart {
                     .attr("marker-end", d => `url(#arrow-${d.from + "_" + d.to})`)
             },
             linkUpdate: function (d, i, arr) {
-                d3.select(this)
-                    .attr('stroke', d => d.data._upToTheRootHighlighted ? attrs.highlightedLinkColor : attrs.linkColor)
-                    .attr('stroke-width', d => d.data._upToTheRootHighlighted ? '2' : '1')
-                if (d.data._upToTheRootHighlighted) {
-                    d3.select(this).raise()
+                if (d.data.parentId === dummyRootNodeId) {
```

Comment:
now that we have the source code let's expose a attr to control this

.hiddenNodeIds([id1, id2...])

And then we can use that as well to hide clicks and other things